lib/theme.js

```diff
@@ -145,6 +148,15 @@ export default {
       color: 'inherit'
     }
   },
+  alerts: {
+    ...theme.alerts,
+    information: {
+      variant: 'alerts.primary',
+      borderColor: 'info',
+      bg: 'infoAlt',
+      color: 'onInfo'
+    }
+  },
```

Comment:
I think info and notice are the same thing. Check the "Badges and Alerts" section here: https://dai-ui.vercel.app/